access-control-app/src/cmr/access_control/services/acl_service.clj

```diff
@@ -390,6 +408,7 @@
          :items
          (map :s3-bucket-and-object-prefix-names)
          flatten
+         parse-single-string-multi-valued-bucket-lists
          distinct
          (remove nil?))))
```

Comment:
The `(remove nil?)` gets applied after `parse-single-string-multi-valued-bucket-lists` - is it possible a nil value can be passed to the latter, causing it to choke on the `str/trim` on line 371?